### PR TITLE
Fixes to site

### PR DESCRIPTION
**Pull request by IMG-PRCSNG**
The PR consists of the following changes

Build
- Use Sass dependency through npm
- mdtohtml script is not needed after adding the necessary CSS classes

Scroll
- Scroll behaviour: Clicking the arrow on the pages results in smooth scroll through the css attribute. Removed the JS code.

General
- Add Contact Us section at the end of every page
- Change CSS classes in Contact Us & Portfolio pages to make sure spacing is right on small devices
- Fix broken links that linked to internal pages / sections
- Change text color on buttons on home page for contrast
- Fix footer height and layout of elements
- Fix grid misalignment in [Our Services](https://dreamingspires.dev/our_services/#client_desk_2)
- Fix spacing on page title / subtitle
- Increase font size of body content
- Add more space around section arrows

### CODE REVIEW

**Comment by IMG-PRCSNG** [comment]
Updates

Signup Partial
- Change color of email and add underlines to link text

Footer Partial
- Add underline to all links to show it is a clickable link

Landing Partial
- Update partial with gradient on image
- Use it in all pages, add arrow only if input parameter is non-empty

Portfolio
- Use the landing partial
- Added a section with title / subtitle
- Individual pages - restrict width to 960px

General
- Make title / subtitle size and spacing even across pages

**Comment by IMG-PRCSNG** [comment]
Updated timeline to look like this

localhost_9000_

**Comment by mark-todd** [comment]
Looks good!
First thought:
image

**Comment by mark-todd** [comment]
Second thought:
image

File: content/_index.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 params:
+  description: "DreamingSpires is a company focused on bringing programming talent to Research Organisations in need of them."
```
Comment (mark-todd) [inline content/_index.html]: DreamingSpires -> Dreaming Spires

**Pull request merged**